libmscore/scale.h

```diff
@@ -0,0 +1,50 @@
+//=============================================================================
+//  MuseScore
+//  Music Composition & Notation
+//
+//  Copyright (C) 2002-2012 Werner Schweer
+//
+//  This program is free software; you can redistribute it and/or modify
+//  it under the terms of the GNU General Public License version 2
+//  as published by the Free Software Foundation and appearing in
+//  the file LICENCE.GPL
+//=============================================================================
+
+#ifndef __SCALE_H__
+#define __SCALE_H__
+
+/**
+ \file
+ Definition of Scale class.
+ */
+
+#include "note.h"
+
+namespace Ms {
+
+class Scale {
+public:
+      bool loadScale(const QString&);
+      double getTuning(const Note* note) const;
+
+private:
+      QString getNextLine(QTextStream& in);
+
+      std::vector<double> originalNotes;
```

Comment:
mgavioli, would you mind getting on irc to talk about this diff?

I think your suggestion is for having the keys inputted in the order of the TPC enum, instead of the Scala file format, which I did not implement. I still think I'd like to keep the Scala format. If you find there is a case for having a sharp note higher than the next bemol, you could always change the scala file and input it.

originalNotes will be used for showing the scale and saving it to the file.